css/style.css

```diff
@@ -0,0 +1,230 @@
+/* Busra's Cake Company */
+
+/* grid-container */
+.grid-container {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-template-rows: 120px auto 100px;
+  grid-gap: 10px;
+  background-color: ivory;
+}
+
+/* header */
+header {
+  background-color: rgb(252, 204, 213);
+  display: grid;
+  grid-template-columns: repeat(8, 1fr);
+  grid-template-rows: 50% 50%;
+  grid-column: 1 / -1;
+
+  padding: 10px;
+}
+
+header p {
+  grid-column: 5 / -1;
+  grid-row: 2 / 3;
+  text-align: right;
+  font-style: oblique;
```

Comment:
It'd be good to override the default fonts - they are not visually pleasant, especially on mobile devices - as you did in the previous week's exercise.